File: Scrapper/src/generator/Accomodation/hotel.py
import random as rand


class Hotel:
    def __init__(self, name=None, country=None, city=None, region=None, rooms=None, rating=None, amenities=None):
        self.name = name
        self.country = country
        self.city = city
        self.region = region
        if rating is None:
            rating = rand.randint(1, 5)
        self.rating = rating
        if amenities is None:
            amenities = []
        self.amenities = amenities
        if rooms is None:
            rooms = []
        self.rooms = rooms

File: services/src/TourOperator/Models/Api/Hotel.cs
namespace TourOperator.Models.Api;

public class Hotel
{
	public string Name { get; set; }
	
	public string Country { get; set; }
	
	public string City { get; set; }
	
	public string Region { get; set; }
	
	public int Rating { get; set; }
	
	public List<string> Amenities { get; set; }
	
	public List<Room> Rooms { get; set; }
}
